.github/workflows/gh-pages.yml

```diff
@@ -39,13 +39,6 @@ jobs:
           TAG_NAME="${GITHUB_REF##*/}"
           echo "::set-output name=tag_name::${TAG_NAME}"
 
-      # Build some internal docs and inject a banner on top of it.
-      - name: Build the internal docs
-        run: cargo xtask doc --internal
```

Comment:
Note that building docs is tested by the Netlify build, which does report if it fails. We could consider making a `test-docs-build` CI job just to make it simpler to see if docs build succeeded/failed.